code/modules/clothing/spacesuits/void/void.dm

```diff
@@ -132,31 +133,36 @@
 			helmet.pickup(H)
 			helmet.canremove = 0
 			H << "<span class='info'>You deploy your suit helmet, sealing you off from the world.</span>"
+			playsound(src.loc, 'sound/weapons/guns/interact/pistol_magin.ogg', 75, 1)
 	helmet.update_light(H)
 
 /obj/item/clothing/suit/space/void/verb/eject_tank()
 
 	set name = "Eject Voidsuit Tank"
 	set category = "Object"
-	set src in usr
+	set src in view()
+
+	if(!isliving(usr))
+		return
 
-	if(!isliving(src.loc))
+	if (get_dist(usr, get_turf(src)) > 1)
+		usr << "<span class='warning'>You're too far away to eject the tank.</span>"
 		return
 
 	if(!tank)
-		usr << "There is no tank inserted."
+		usr << "<span class='warning'>There is no tank inserted.</span>"
```

Comment:
It's better to use macro `SPAN_WARNING(...)` instead `<span class='warning'>...</span>`